vector/src/main/java/im/vector/app/features/location/live/map/LocationLiveMapViewFragment.kt

```diff
@@ -50,7 +53,8 @@ import javax.inject.Inject
  */
 class LocationLiveMapViewFragment @Inject constructor(
```

Comment:
When creating the `SymbolManager` for the map in `setupMap()` method, we can add option to allow override of symbols in order to see every users even if they are very near from each other:
```
symbolManager = SymbolManager(mapFragment.view as MapView, mapboxMap, style).also {
    it.iconAllowOverlap = true
}
```